README.md

```diff
@@ -49,8 +49,12 @@ instructions using GitPod.
 Using the [hoprd npm package][6]:
 
 ```sh
-npm install -g @hoprnet/hoprd
-hoprd --admin
+mkdir MY_NEW_HOPR_TEST_FOLDER
+cd MY_NEW_HOPR_TEST_FOLDER
+yarn add @hoprnet/hoprd@1.72.11
```

Comment:
Are we now asking users to install `yarn`? Remembering us use `npm install @hoprnet/hoprd@x.y.z`

Reply:
good point

Reply:
shouldn't it be `npm install -g XYZ` ?

Reply:
fixed

Reply:
we want to move away from global installations, has been a cause of many problems